redhat-cop/casl-ansible
# Updated DNS server role + example playbook

#### What does this PR do?
Updated the DNS server role with a few fixes/clean-up + added an example dns playbook

#### How should this be manually tested?
- Run the example playbook to configure an existing VM/instance as a dns server
- Run the full CASL provisioning to ensure nothing else broke

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
